.env.sample

```diff
@@ -0,0 +1,4 @@
+# Gatsby requires a prefix of GATSBY and React requires a prefix of REACT_APP
+# Gatsby doc https://www.gatsbyjs.com/docs/how-to/local-development/environment-variables/
+# React doc https://create-react-app.dev/docs/adding-custom-environment-variables/
+GATSBY_REACT_APP_GCAL_API_KEY=AIzaSyCBklATFMNjWUjJVZswlTmoyZh27FbaHDQ
```

Comment:
Just to confirm, this is not a real key, right?

Reply:
It is a real key. It’s my personal one that needs to be replaced once all the business logic and UI are approved